pkg/scheduler/framework/v1alpha1/framework.go

```diff
@@ -345,16 +345,17 @@ func (f *framework) RunPrebindPlugins(
 	pc *PluginContext, pod *v1.Pod, nodeName string) *Status {
 	for _, pl := range f.prebindPlugins {
 		status := pl.Prebind(pc, pod, nodeName)
-		if !status.IsSuccess() {
-			if status.Code() == Unschedulable {
-				msg := fmt.Sprintf("rejected by %v at prebind: %v", pl.Name(), status.Message())
-				klog.V(4).Infof(msg)
-				return NewStatus(status.Code(), msg)
-			}
-			msg := fmt.Sprintf("error while running %v prebind plugin for pod %v: %v", pl.Name(), pod.Name, status.Message())
-			klog.Error(msg)
-			return NewStatus(Error, msg)
+		if status.IsSuccess() {
+			continue
```

Comment:
I have never liked `continue` and I think its usage should be limited to the cases were there is no clean alternative. Granted, unlike `goto`, it does not have the horrible effects of ruining ASTs/DAGs of compilers, but from my point of view it feels like `goto` in certain ways. In this example, the body of the loop below `if status.IsSuccess()` is not under any `if` statement, but "in fact" it is, you just do not notice it if you miss the above `if` statement. In this case the body is small and the loop is not nested. If there were a couple of nested loops with larger body, a `continue` could confuse readers.

Reply:
I updated the PR with the non-continue approach, please take another look.